packages/amplify-migration-tests/src/__tests__/migration_tests/transformer_migration/api.key.migration.test.ts

```diff
@@ -13,15 +23,36 @@ describe('amplify key force push', () => {
     deleteProjectDir(projRoot);
   });
 
-  it('init project, add key and migrate with force push', async () => {
+  // it('init project, add key and migrate with force push', async () => {
```

Comment:
should this test still be commented out? If so, use it.skip() or remove it entirely